distro/pom.xml

```
@@ -20,6 +20,7 @@
     <maven.compiler.source>8</maven.compiler.source>
     <maven.compiler.target>8</maven.compiler.target>
     <PatientIdentifierGeneratorVersion>0.1.0</PatientIdentifierGeneratorVersion>
+    <MSFReferenceapplicationDockerVersion>qa</MSFReferenceapplicationDockerVersion>
```

Based on context, this is the RefApp docker version and not a custom image published under MSF Org

```suggestion
    <ReferenceApplicationDockerVersion>qa</ReferenceApplicationDockerVersion>
```